Example/src/App.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import {ScrollView, StyleSheet, Text, View} from 'react-native';
+import QRCode, {QRCodeProps} from 'react-native-qrcode-svg';
+import SVG from './assets/ruby.svg';
+
+type QRCodeWithDescriptionProps = {
+  text: string;
+  qrCode: QRCodeProps;
+};
+
+const QRCodeWithDescription = ({text, qrCode}: QRCodeWithDescriptionProps) => (
+  <View style={styles.scrollViewElement}>
+    <Text style={styles.text}>{text}</Text>
+    <QRCode {...qrCode} />
+  </View>
+);
+
+const styles = StyleSheet.create({
```

Comment:
Shouldn't the `styles` be at the bottom of the file?

Reply:
Moved them to be just above the `App` component. Is it the right place?

Reply:
I've moved them to a separate file. Is that ok?

Reply:
Yeah, it's up to you 😅